packages/template-retail-react-app/app/components/login/index.jsx

```diff
@@ -17,9 +17,13 @@ import {
 } from '@salesforce/retail-react-app/app/components/shared/ui'
 import {AlertIcon, BrandLogo} from '@salesforce/retail-react-app/app/components/icons'
 import LoginFields from '@salesforce/retail-react-app/app/components/forms/login-fields'
+import SocialLogin from '@salesforce/retail-react-app/app/components/social-login'
 import {noop} from '@salesforce/retail-react-app/app/utils/utils'
+import {getConfig} from '@salesforce/pwa-kit-runtime/utils/ssr-config'
 
 const LoginForm = ({submitForm, clickForgotPassword = noop, clickCreateAccount = noop, form}) => {
+    const idps = getConfig().app?.login?.idps
```

Comment:
should we include the changes to `default.js` with idps set to Apple and Google?

Reply:
I'm actually not sure what default state we want. I was going to figure that out when we hook up the real code to the buttons